backend/pycon/settings/base.py

```diff
@@ -76,15 +77,17 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [root("frontend")],
+        "DIRS": [],
         "APP_DIRS": True,
         "OPTIONS": {
             "context_processors": [
                 "django.template.context_processors.debug",
                 "django.template.context_processors.request",
                 "django.contrib.auth.context_processors.auth",
                 "django.contrib.messages.context_processors.messages",
-            ]
+                "custom_admin.context_processors.admin_settings",
+            ],
+            "debug": True,
```

Comment:
Maybe it will better to use `DEBUG` environment variable here instead of a `True` as default ?
Something like that:
```suggestion
            "debug": env("DEBUG", default=False)
```

Reply:
The debug here doesn't seem to do anything useful so I will just remove it